### PR TITLE
UI partially finished

### PR DESCRIPTION
**Pull request by anmol-lf**
@bishalg , @leapfrog-saugat , @kusalshrestha , @sanjaymhj , @vsujan , @laxmankhanal

### CODE REVIEW

File: Vaccination/Utilities/DateUtils.swift [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    return age
+}
+
+func getCurrentDate() -> NSDate {
+    let date = NSDate()
+    return date
+}
+
+func parseDate(dateStr: String, format: String="yyyy-MM-dd") -> NSDate {
+    let dateFmt = NSDateFormatter()
```
Comment (vsujan) [inline Vaccination/Utilities/DateUtils.swift]: naming seems odd

File: Vaccination/Utilities/FileUtils.swift [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import Foundation
+import UIKit
+
+func saveFile(withImage image: UIImage, withFilename filename: String) -> Bool {
+    let fileManager = NSFileManager.defaultManager()
+    if let image1: UIImage = image {
```
Comment (vsujan) [inline Vaccination/Utilities/FileUtils.swift]: "image1" looks odd

File: Vaccination/Utilities/DateUtils.swift [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+func calculateAge(withDOB dob: NSDate) -> Double {
+    let currentDate: NSDate = NSDate()
+    let age = floor(currentDate.timeIntervalSinceDate(dob))/(60*60*24)//Time from DOB(In Days)
+    return age
+}
+
+func getCurrentDate() -> NSDate {
+    let date = NSDate()
+    return date
```
Comment (kusalshrestha) [inline Vaccination/Utilities/DateUtils.swift]: No use of saving NSDate to new variable.. Can be returned directely

File: Vaccination/Utilities/DateUtils.swift (hunk 1 above)
Comment (anmol-lf) [inline Vaccination/Utilities/DateUtils.swift]: :ok:

File: Vaccination/Utilities/FileUtils.swift (hunk 2 above)
Comment (anmol-lf) [inline Vaccination/Utilities/FileUtils.swift]: :ok:

File: Vaccination/Utilities/DateUtils.swift (hunk 3 above)
Comment (anmol-lf) [inline Vaccination/Utilities/DateUtils.swift]: :ok:

**Pull request merged**